fe/fe-core/src/main/java/com/starrocks/sql/parser/StarRocks.g4

```diff
@@ -2675,4 +2693,5 @@ nonReserved
     | WARNINGS | WEEK | WHITELIST | WORK | WRITE  | WAREHOUSE | WAREHOUSES
     | YEAR
     | DOTDOTDOT | NGRAMBF
+    | FIELD
```

Comment:
also add ARRAR_ELEMENT

Reply:
> also add ARRAR_ELEMENT

Done, changed in https://github.com/StarRocks/starrocks/commit/0003760e4df6dbf046756b8683af4a9a48ff75b9